streams/src/test/java/org/apache/kafka/streams/state/internals/CachingPersistentSessionStoreTest.java

```diff
@@ -258,58 +266,45 @@ public void shouldPutBackwardFetchAllKeysFromCache() {
     @Test
     public void shouldCloseWrappedStoreAndCacheAfterErrorDuringCacheFlush() {
         setUpCloseTests();
-        EasyMock.reset(cache);
-        cache.flush(CACHE_NAMESPACE);
-        EasyMock.expectLastCall().andThrow(new RuntimeException("Simulating an error on flush"));
-        EasyMock.replay(cache);
-        EasyMock.reset(underlyingStore);
-        underlyingStore.close();
-        EasyMock.replay(underlyingStore);
+        doThrow(new RuntimeException("Simulating an error on flush")).doNothing().when(cache).flush(anyString());
 
         assertThrows(RuntimeException.class, cachingStore::close);
-        EasyMock.verify(cache, underlyingStore);
+        verify(cache).flush(eq(CACHE_NAMESPACE));
+        verify(cache).close(eq(CACHE_NAMESPACE));
+        verify(underlyingStore).close();
     }
 
     @Test
     public void shouldCloseWrappedStoreAfterErrorDuringCacheClose() {
         setUpCloseTests();
-        EasyMock.reset(cache);
-        cache.flush(CACHE_NAMESPACE);
-        cache.close(CACHE_NAMESPACE);
-        EasyMock.expectLastCall().andThrow(new RuntimeException("Simulating an error on close"));
-        EasyMock.replay(cache);
-        EasyMock.reset(underlyingStore);
-        underlyingStore.close();
-        EasyMock.replay(underlyingStore);
+
```

Comment:
nit: Please remove this line here and in other parts of this PR. Stubbing is part of the setup IMO.

Reply:
If you are referring to `setUpCloseTests` - it isn't used in all tests in the class, so I suspect the original author abstracted it in a method for ease-of-use. I don't think it will aid readability if we try to move it in the `@Before` method.